_data/simple-icons.json

```
@@ -4228,6 +4228,11 @@
 		"hex": "3793EF",
 		"source": "https://deepnote.com"
 	},
+	{
+		"title": "Deepseek",
```

Wrong spelling. It should be `DeepSeek`.

```suggestion
		"title": "DeepSeek",
```